examples/routers/switch/index.vue

```diff
@@ -28,6 +28,16 @@ import Icon from '@/icon'
 import ACol from '@/col'
 import AtuSwitch from '@/switch'
 export default {
+  data () {
+    return {
+      check: false
+    }
+  },
+  methods: {
+    canChange (value) {
```

Comment:
method name should be handleChange, sense is better than canXXXX which sounds like a props.

Reply:
em... these code in example, so let it go...